runtime/src/accounts_background_service.rs

```diff
@@ -418,6 +418,24 @@ impl SnapshotRequestHandler {
         );
         Ok(snapshot_root_bank.slot())
     }
+
+    /// Returns the slot of the next snapshot request to be handled
+    fn peek_next_snapshot_request_slot(&self) -> Option<Slot> {
```

Comment:
i was going to comment about "peek" usually being side-effect free, but then i looked under the hood and realized it's all side-effects down there and we're just keeping the status quo